include/thread_pool.h

```diff
@@ -29,6 +29,7 @@
 
 #include <sys/types.h>
 #include <pthread.h>
+#include <limits.h>
```

Comment:
The only `PTHREAD_*` construct from limits.h used across the project is
```
./lib/libtpool/thread_pool.c:           minstack = PTHREAD_STACK_MIN;
./lib/libzpool/kernel.c:#define TS_STACK_MIN    MAX(PTHREAD_STACK_MIN, 32768)
```
It's much better to `#include <limits.h>` in the respective `*.c` files than in the header.

Reply:
Updated files